util/src/main/java/tc/oc/pgm/util/bukkit/EntityTypes.java

```diff
@@ -34,7 +34,17 @@ public class EntityTypes {
   public static final EntityType ARMOR_STAND = EntityType.ARMOR_STAND;
   public static final EntityType MINECART_COMMAND =
       parse("MINECART_COMMAND", "COMMAND_BLOCK_MINECART");
-  public static final EntityType BOAT = EntityType.BOAT;
+  public static final EntityType BOAT = parse(
+      "BOAT",
+      "ACACIA_BOAT",
+      "BIRCH_BOAT",
+      "CHERRY_BOAT",
+      "DARK_OAK_BOAT",
+      "JUNGLE_BOAT",
+      "MANGROVE_BOAT",
+      "OAK_BOAT",
+      "PALE_OAK_BOAT",
+      "SPRUCE_BOAT");
```

Comment:
Yeah... this is not an "all of these" kinda situation, only the first to match will be considered the real mapping, so in modern the boat entity type just means acacia boat, this is why kill-entities with this entity type doesn't work.

If you want kill-entities to work you'd have to specify each individual boat in the kill-entities, or have a different/higher concept of "entity matcher" where boat can mean `*_BOAT` in modern, similar to what we do with blocks (ie: "WOOL" is taken to mean "WHITE_WOOL", "RED_WOOL", etc, as part of material matcher)